src/components/map/popups/MapPopupPilot.vue

```diff
@@ -59,6 +59,14 @@
                 />
             </div>
         </template>
+        <template
+            v-if="vatGlassesActive"
+            #tab-atc
+        >
+            <common-notification cookie-name="vatglasses-atc-warning">
+                This data may not be reliable when VatGlasses integration is active. Refer to map instead.
```

Comment:
In Firefox there is no automatic linebreak, it makes the field scrollable to the right

Reply:
Oh.